programs/ssl/ssl_server2.c

```diff
@@ -3363,9 +3363,9 @@ int main( int argc, char *argv[] )
         }
         else
 #endif /* MBEDTLS_USE_PSA_CRYPTO */
-        if( ( ret = mbedtls_ssl_conf_psk( &conf, psk, psk_len,
+        if( ( psk_len != 0 ) && ( ( ret = mbedtls_ssl_conf_psk( &conf, psk, psk_len,
```

Comment:
Why not `psk_len > 0` to be consistent with `ssl_client2`? Can `psk_len` even be negative?

Reply:
updated